BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -115,7 +127,7 @@ def main():
 
     openssl_path = get_openssl_path()
 
-    if args.KeySize is 'ALL':
+    if args.KeySize == 'ALL':
         # Generate keys for key size 2048 and 3072
         GenerateRsaPrivKeys(openssl_path, args.KeyDir, RSA_KEY_SIZE["RSA2048"])

```

Comment:
can we use camel case to make coding style consistent?   generate_rsa_priv_keys ()/ generate_rsa_pub_keys()